dev/com.ibm.ws.jdbc.metatype/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -81,7 +81,7 @@
   <AD id="enableBeginEndRequest"                  name="internal"      description="internal use only"  ibmui:group="Advanced" required="false" type="Boolean"/>
   <AD id="enableConnectionCasting"                name="%enblConCast"  description="%enblConCast.desc"  ibmui:group="Advanced" type="Boolean" default="false"/>
   <AD id="heritage"                               name="internal"      description="internal use only"  ibmui:group="Advanced" required="false" type="String"  cardinality="1" ibm:type="pid" ibm:reference="com.ibm.ws.jdbc.dataSource.heritage" ibm:flat="true"/>
-  <AD id="identifyException"                      ibm:beta="true"                                       ibmui:group="Advanced" required="false" type="String"  cardinality="1000" ibm:type="pid" ibm:reference="com.ibm.ws.jdbc.dataSource.identifyException" ibm:flat="true"/>
+  <AD id="identifyException"                                                                            ibmui:group="Advanced" required="false" type="String"  cardinality="1000" ibm:type="pid" ibm:reference="com.ibm.ws.jdbc.dataSource.identifyException" ibm:flat="true"/>
```

Comment:
PB failed ... I think it is complaining that this element does not have name and description

Reply:
It might be complaining incorrectly because the generated schema looked correct regardless of whether name/description are present on this attribute (it gets the name/description from the flattened config element).  However, it should be possible to work around this by including that same information here as well.  The generated schema appears to be the same either way.  Trying a new personal build with this update...